dotCMS/src/integration-test/java/com/dotcms/ema/EMAWebInterceptorTest.java

```diff
@@ -0,0 +1,110 @@
+package com.dotcms.ema;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import com.dotcms.datagen.SiteDataGen;
+import com.dotcms.datagen.TestUserUtils;
+import com.dotcms.ema.proxy.MockHttpCaptureResponse;
+import com.dotcms.filters.interceptor.Result;
+import com.dotcms.security.apps.AppSecrets;
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.beans.Host;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotSecurityException;
+import com.dotmarketing.util.PageMode;
+import com.dotmarketing.util.WebKeys;
+import com.liferay.portal.model.User;
+import java.io.IOException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class EMAWebInterceptorTest {
+
+    private static HttpServletRequest request = mock(HttpServletRequest.class);
+    private static HttpServletResponse response = mock(HttpServletResponse.class);
+    private static HttpSession session = mock(HttpSession.class);
+    private static EMAWebInterceptor emaWebInterceptor;
+    private static User admin;
+
+    @BeforeClass
+    public static void prepare() throws Exception {
```

Comment:
Codacy found an issue: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5922984)